### DISCUSSION TITLE
Disable Dark / Light Mode Toggle and Set Default Mode

### DISCUSSION THREAD

**Discussion by selected-pixel-jameson** [vuepress/core#183] [Q&A] [answered]
Is there a way to disable the dark / light mode toggle and set the default mode to `light`?

**Comment by vmihailenco** [comment]
@meteorlxy gentle ping. Would be nice to have an option to remove/hide the dark mode button.

**Comment by meteorlxy** [comment, marked as answer]
Supported in 6517ce1

**Reply by guumaster** [reply]
Can the initial theme colors be forced to dark when `darkMode` is disabled?

**Reply by meteorlxy** [reply]
You can add `class="dark"` to `<html/>` element manually, or change the CSS variables youself.
